sparkle/package.json

```diff
@@ -77,7 +77,7 @@
   },
   "homepage": "https://github.com/dust/dust-tt#readme",
   "dependencies": {
-    "@esbuild/linux-x64": "^0.20.0",
+    "@esbuild/linux-x64": "^0.18.20",
```

Comment:
this line should be killed, idem come to check that, there is a sequence of things to do since the changes that were made on main this morning